Sources/SKCore/BuildServerBuildSystem.swift

```diff
@@ -57,6 +57,15 @@ public final class BuildServerBuildSystem {
     }
   }
 
+  deinit {
+    do {
+      _ = try self.buildServer?.sendSync(ShutdownBuild())
```

Comment:
Does this need to be synchronous? Seems like you could do the error logging and exit notification from inside the completion block of an async request.